custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/README.md

```diff
@@ -0,0 +1,167 @@
+# Custom Metrics with Prometheus and Stackdriver
+
+This is a minimal example showing how to export custom metrics using 
+[prometheus text format](https://prometheus.io/docs/instrumenting/exposition_formats/) to Stackdriver
+in a k8s cluster running on GCE or GKE. 
+
+## Prometheus dummy exporter
+
+A simple prometheus-dummy-exporter component exposes a single prometheus metric 
+with a constant value. The metric name and value can be passed in via flags.
+There is also a flag to configure the port on which the metrics are served. 
+
+## Prometheus to Stackdriver sidecar container
+
+Prometheus Dummy exporter is deployed with a sidecar container running 
```

Comment:
Please remove the repetition ("sidecar [Prometheus to Stackdriver] container")

Reply:
Done.